apps/opik-backend/src/main/java/com/comet/opik/infrastructure/EncryptionService.java

```diff
@@ -0,0 +1,61 @@
+package com.comet.opik.infrastructure;
+
+import com.google.inject.ImplementedBy;
+import jakarta.inject.Inject;
+import jakarta.inject.Singleton;
+import lombok.NonNull;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.SecretKeySpec;
+
+import java.nio.charset.StandardCharsets;
+import java.security.Key;
+import java.util.Base64;
+
+@ImplementedBy(EncryptionServiceImpl.class)
+public interface EncryptionService {
+    String encrypt(String data);
+    String decrypt(String encryptedData);
+}
+
+@Singleton
+class EncryptionServiceImpl implements EncryptionService {
+
+    private static final String ALGO = "AES";
+    private final Base64.Encoder mimeEncoder = Base64.getMimeEncoder();
+    private final Base64.Decoder mimeDecoder = Base64.getMimeDecoder();
+    @NonNull private final Key key;
+
+    @Inject
+    public EncryptionServiceImpl(@NonNull OpikConfiguration config) {
+        byte[] keyBytes = config.getEncryption().getKey().getBytes(StandardCharsets.UTF_8);
+        key = new SecretKeySpec(keyBytes, ALGO);
+
+        System.out.println("Conf1: " + config.getEncryption().toString());
```

Comment:
This should be a debug log. But I'd rather remove it, even if you're excluding the key from the toString method.

Reply:
Removed, leftovers from manual testing which I forgot to delete.